clusterloader2/pkg/dependency/dra/dra.go

```diff
@@ -34,7 +34,7 @@ import (
 
 const (
 	draDependencyName      = "DRATestDriver"
-	draNamespace           = "dra-example-driver"
+	draNamespace           = "dra-driver"
```

Comment:
I dont think we should change the name of the driver. 

The "dra-example-driver" has a very clear reference to this: https://github.com/kubernetes-sigs/dra-example-driver/tree/main.

Are there any other reasons why we need to change the name here that I am missing?